DMCompiler/DM/Visitors/DMVisitorExpression.cs

```diff
@@ -321,6 +327,12 @@ public void VisitIdentifier(DMASTIdentifier identifier)
             Result = new Expressions.ModulusAssign(modulusAssign.Location, lhs, rhs);
         }
 
+        public void VisitModulusModulusAssign(DMASTModulusModulusAssign modulusModulusAssign) {
+            var lhs = DMExpression.Create(_dmObject, _proc, modulusModulusAssign.A);
+            var rhs = DMExpression.Create(_dmObject, _proc, modulusModulusAssign.B);
```

Comment:
You can do `a %%= new`
```suggestion
            var lhs = DMExpression.Create(_dmObject, _proc, modulusModulusAssign.A, _inferredPath);
            var rhs = DMExpression.Create(_dmObject, _proc, modulusModulusAssign.B, lhs.Path);
```